tests/fixtures/forms.html

```diff
@@ -4,6 +4,7 @@
     <script src="https://cdn.tailwindcss.com"></script>
   </head>
   <body class="page">
+    <!-- CAUSES A REDIRECT -- IF HAVE PATH THAT HAS BOTH POST AND GET, POST ISN'T CACHED OR GET ISN'T USED FOR POST RESPONSE -->
```

Comment:
Maybe reads better like so:
```suggestion
    <!-- CAUSES A REDIRECT -- IF WE HAVE PATH THAT HAS BOTH POST AND GET, TEST THAT POST ISN'T CACHED AND GET ISN'T USED FOR POST RESPONSE -->
```
And super duper minor but are all caps necessary?

Reply:
Sounds good!

> And super duper minor but are all caps necessary?

Haha, good catch! This was originally a "note to self" hence the capitalization, but it won't be a "note to self" anymore 😆